index.ts

```diff
@@ -0,0 +1,11 @@
+export { FormComponent } from './src/form.component';
+export { FormElementComponent } from './src/formelement.component';
+export { WidgetChooserComponent } from './src/widgetchooser.component';
+export { WidgetRegistry } from './src/widgetregistry';
+export { Validator } from './src/model/validator';
+export { DefaultWidgetRegistry } from './src/defaultwidgets/defaultwidgetregistry';
+
+
+export * from './src/widget';
```

Comment:
I usually prefer to avoid `export *` (unless for a root index.ts export the full /dist folder or something similar).